### PR TITLE
add helpcenter link

### PR DESCRIPTION
**Pull request by mmilenkovic**

### CODE REVIEW

File: src/views/home/components/report-issue-button/report-issue-button.styles.js [hunk 1]
```diff
@@ -9,11 +9,13 @@ const reportIssueButtonStyles = createUseStyles((theme) => ({
...
   text: {
-    width: '100%',
-    textAlign: 'center',
     fontWeight: theme.fontWeights.medium,
     color: theme.palette.grey.main,
     padding: `${theme.spacing(1.5)}px 0`
+  },
+  separator: {
+    color: theme.palette.grey.main,
+    padding: theme.spacing(1.5)
```
Comment (elias-garcia) [inline src/views/home/components/report-issue-button/report-issue-button.styles.js:18]: I think that the separator only needs to have padding on the sides right? Like: `${theme.spacing(1.5)}px 0`

File: src/views/home/components/report-issue-button/report-issue-button.view.jsx [hunk 2]
```diff
@@ -16,6 +16,15 @@ function ReportIssueButton () {
       >
         Report an issue
       </a>
+      <span className={classes.separator}> | </span>
```
Comment (elias-garcia) [inline src/views/home/components/report-issue-button/report-issue-button.view.jsx:19]: Are they blank spaces needed if it already has padding?

File: src/views/home/components/report-issue-button/report-issue-button.styles.js (hunk 1 above)
Comment (mmilenkovic) [reply src/views/home/components/report-issue-button/report-issue-button.styles.js:18]: Nope it needs both:

Screenshot 2021-04-29 at 15 34 17
Screenshot 2021-04-29 at 15 34 23

File: src/views/home/components/report-issue-button/report-issue-button.view.jsx (hunk 2 above)
Comment (mmilenkovic) [reply src/views/home/components/report-issue-button/report-issue-button.view.jsx:19]: Correct, not needed. Removed.

**Review by elias-garcia — APPROVED** [review]

**Pull request merged**